src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISDeployer.cs

```diff
@@ -26,44 +27,46 @@ public IISDeployer(DeploymentParameters startParameters, ILogger logger)
         {
         }
 
-        public override DeploymentResult Deploy()
+        public override async Task<DeploymentResult> DeployAsync()
```

Comment:
I can now that we have a smoother plan for larger breaking changes. I wasn't confident at the time that nobody uses it